rust/Earthfile

```diff
@@ -70,43 +91,41 @@ get-jq:
     FROM alpine:3.18.3
     ARG USERARCH
     ARG version=1.7
-    RUN wget -O jq https://github.com/jqlang/jq/releases/download/jq-${version}/jq-linux-${USERARCH} && \
+    RUN wget -O jq https://github.com/jqlang/jq/releases/download/jq-${version}/jq-linux-${USERARCH}; \
         chmod +x jq
     SAVE ARTIFACT jq
 
 # INSTALL_CARGO_SWEEP installs cargo-sweep if it doesn't exist already.
 INSTALL_CARGO_SWEEP:
     COMMAND
-    DO +RUN_WITH_CACHE --command="set -e;
-        if [ ! -f \$CARGO_HOME/bin/cargo-sweep ]; then
-          cargo install cargo-sweep --root \$CARGO_HOME;
-        fi;"
+    RUN if [ ! -f $CARGO_HOME/bin/cargo-sweep ]; then \
+          echo "Installing cargo sweep" ; \
+          cargo install cargo-sweep --root $CARGO_HOME; \
+        fi;
```

Comment:
just curious why you used a `RUN if` instead of `IF ... RUN`

Reply:
`IF ... RUN` runs two shell instances also, within RUN we just have bash semantics (for the conditional), that I am more familiarized with